### ISSUE TITLE
beta=0.1 used for reference VAE

### ISSUE THREAD

**Issue by eitanrich** [facebookresearch/irmae#1]
The default beta value used for the KL term in the reference VAE is 0.1. Can you explain why this value was chosen (and not 1)?

**Comment by jingli9111** [comment]
We treat it as a hyperparameter and did grid search to find best baseline. 0.1 is the value that gives best result on validation set.
Note the default beta value from original VAE paper should be 0.5 (not 1).

**Comment by gabrieldernbach** [comment]
I can not find the KL term, which I had expected to see close to the line 245 in models.py
```
        if self.vae:
            loss = F.binary_cross_entropy(x_hat, x)
```
Fulltext search of `beta` or `kl` did not show any finds.

Did you remove the regularization in the end?

Also did you compare the vae with an mse loss as well?

**Comment by jingli9111** [comment]
Hi @gabrieldernbach , thanks for pointing this out. It's a mistake in our last update. Here's what it should look like:
```
        if self.vae:
            loss = F.binary_cross_entropy(x_hat, x)
            loss -= self.args.beta * torch.mean(
                1 + logvar - mu.pow(2) - logvar.exp()
            )
```
